src/pages/become-a-sponsor.js

```diff
@@ -248,6 +271,150 @@ export default () => {
+                <span role="img" aria-label="Benefit included">
+                  1
+                </span>
+              </td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  1
+                </span>
+              </td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  1
+                </span>
+              </td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  1
+                </span>
+              </td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  1
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td>Branding in conference media</td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+            </tr>
+            <tr>
+              <td>Branding on t-shirt</td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️✔️
+                </span>
+              </td>
+              <td />
+            </tr>
+            <tr>
+              <td>Conference tickets</td>
+              <td css={styles.highlight}>
+                <span role="img" aria-label="Benefit included">
+                  ️️2
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
+                  ️️2
+                </span>
+              </td>
+              <td>
+                <span role="img" aria-label="Benefit included">
```

Comment:
the aria-label is only needed to wrap the emoji so those using a screenreader know what the check means. We should remove it in the cases where it is wrapping a number because it will override the number and it will lose meaning.